### ISSUE TITLE
PC1: Unhandled exception. System.IndexOutOfRangeException: Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by MikeVapour** [Nenkai/PCarsTools#3]
Hi,

I was trying to unpack PC1 bff's, and I get this error message:

```
Unhandled exception. System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at PCarsTools.Xml.BBinaryFile.GetChunkHeader(UInt32 chunkId) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\BBinaryFile.cs:line 44
   at PCarsTools.Xml.BBinaryFile.GetChunk(UInt32 id) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\BBinaryFile.cs:line 35
   at PCarsTools.Xml.BBinaryFile.GetChunk(String chunkName) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\BBinaryFile.cs:line 30
   at PCarsTools.Xml.BXmlFile.GetHeader() in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Xml\BXmlFile.cs:line 149
   at PCarsTools.Xml.BXmlFile.GetCollection(UInt32 hash) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Xml\BXmlFile.cs:line 238
   at PCarsTools.Xml.BXmlFile.GetCollection(String nodePath) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Xml\BXmlFile.cs:line 233
   at PCarsTools.Xml.BXmlFile.GetNodes(String nodePath) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Xml\BXmlFile.cs:line 137
   at PCarsTools.Config.BConfig.LoadConfig(String fileName) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Config\BConfig.cs:line 35
   at PCarsTools.Program.Pak(PakVerbs options) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Program.cs:line 101
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at PCarsTools.Program.Main(String[] args) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Program.cs:line 25
```

Is it something you can address?
I was trying to unpack PC1 track bff's.

**Comment by Nenkai** [comment]
I will need the `Languages/Languages.bml` file.

**Comment by MikeVapour** [comment]
here you go:
[Languages.zip](https://github.com/Nenkai/PCarsTools/files/9996044/Languages.zip)

**Comment by Nenkai** [comment]
fb2d34a77d6a28a63e3fca7b68efd913ad89dc2a

**Comment by MikeVapour** [comment]
Just tested. I still get errors:
```
Warning - unmatched UID/Hash: tracks\_data\dynamic\physics\dynamic.obj.bml (target=CFF62CF951D87CE5, got=17307067313501638160
Unhandled exception. System.ArgumentOutOfRangeException: Non-negative number required. (Parameter 'value')
   at System.IO.FileStream.set_Position(Int64 value)
   at PCarsTools.BPakFile.UnpackFromStream(BPakFileTocEntry entry, BExtendedFileInfoEntry extEntry, String output) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\BPakFile.cs:line 247
   at PCarsTools.BPakFile.UnpackAll(String outputDir) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\BPakFile.cs:line 198
   at PCarsTools.Program.Pak(PakVerbs options) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Program.cs:line 120
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at PCarsTools.Program.Main(String[] args) in C:\Users\nenkai\source\repos\PCarsTools\PCarsTools\Program.cs:line 25
```

Tried with several files. I can provide a small file whose unpacking produces the error above:
[Summerton_Physics.zip](https://github.com/Nenkai/PCarsTools/files/9998117/Summerton_Physics.zip)

**Comment by Nenkai** [comment]
Addressed in 503bdd56fe779141a9028fee4b423a9ad9a72cba
